freqtrade/commands/arguments.py

```diff
@@ -415,3 +419,9 @@ def _build_subcommands(self) -> None:
                                               parents=[_common_parser])
         webserver_cmd.set_defaults(func=start_webserver)
         self._build_args(optionlist=ARGS_WEBSERVER, parser=webserver_cmd)
+
+        # Add backtesting analysis subcommand
+        analysis_cmd = subparsers.add_parser('analysis', help='Analysis module.',
```

Comment:
I'm not entirely sure i like the "analysis" name.
it's pretty generic - while it's really only analyzing backtest results - so maybe `backtesting-analysis` would be more appropriate, and less missleading?

Reply:
Done